Tactility/Source/settings/SystemSettings.cpp

```diff
@@ -17,6 +20,40 @@ static Mutex mutex;
 static bool cached = false;
 static SystemSettings cachedSettings;
 
+static bool parseEntry(const std::string& input, std::string& outName, std::string& outCode) {
+    std::string partial_strip = input.substr(1, input.size() - 3);
+    auto first_end_quote = partial_strip.find('"');
+    if (first_end_quote == std::string::npos) {
+        return false;
+    } else {
+        outName = partial_strip.substr(0, first_end_quote);
+        outCode = partial_strip.substr(first_end_quote + 3);
+        return true;
+    }
+}
+
+static std::string findTimeZoneName(const std::string& code) {
+    auto path = std::string(file::MOUNT_POINT_SYSTEM) + "/timezones.csv";
+    auto* file = fopen(path.c_str(), "rb");
+    if (file == nullptr) {
+        TT_LOG_E(TAG, "Failed to open %s", path.c_str());
+        return "";
+    }
+    char line[96];
+    std::string name;
+    std::string parsedCode;
+    while (fgets(line, 96, file)) {
+        if (parseEntry(line, name, parsedCode)) {
+            if (parsedCode == code) {
+                fclose(file);
+                return name;
+            }
+        }
+    }
+    fclose(file);
+    return "";
+}
```

Comment:
I mistakenly thought these were now also used by the related app (when we last talked about these changes on Discord). They are not, so the changes in this file can be reverted.